cpp/src/slice2swift/SwiftUtil.h

```diff
@@ -8,94 +8,78 @@
 
 using StringPairList = std::list<std::pair<std::string, std::string>>;
 
-namespace Slice
+namespace Slice::Swift
 {
-    const int TypeContextInParam = 1;
-    const int TypeContextProtocol = 2;
-
     std::string getSwiftModule(const ModulePtr&, std::string&);
     std::string getSwiftModule(const ModulePtr&);
 
```

Comment:
Every single function in this class was effectively static, and the class held no data.
So I turned it into what it logically was: a nested namespace; there was no reason for it to be a class.